roles/pulp/templates/pulp.container.j2

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Pulp 
+After=network.target
+
+[Container]
+Image=pulp/pulp:stable
+ContainerName=pulp
+PublishPort=8080:80
+Volume=pulp_settings:/etc/pulp
```

Comment:
How does this declaration work? Is that a `pulp_settings` directory relative to the current directory this gets run from?

Reply:
When I tested it, it created a volume that you could also see with `podman volume ls`. You can also define specific volumes with (for example) `pulp_settings.volume`.

Reply:
Yes, This `pulp_settings:/etc/pulp` is a Podman named volume here. We can see it with `podman volume ls`. I can make it more explicit as @ekohl suggested using `.volume` syntax

Reply:
Where on the filesystem does this volume get created? I tried to read up on it but I didn't feel like I found a clear answer.

Reply:
I think podman creates its volumes in `/var/lib/containers`